templates/4a_compensate_image_data_round2.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p>f the user have more than 1 set of compensations done, the user will have to combine the coefficients from different csv files together into one csv. Maybe we should explicitly state that here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Is it better for the user to do this themself, or should be provide a function that does this for them?

Reply:
Yup you can just add all the CSVs but the user has to select which one from each compensation set. For example if the user tested 0.5X,1X,1.5X for a set and decided to go with the 0.5X and in another set of input/output channels selected 1X then only those 2 gets into the final CSV.

Reply:
@HPiyadasa @ngreenwald on second thought, I'm going to branch a PR off of this one to address this, this PR itself is getting big and adding quite a bit more to the design doc.

I can open a separate issue for this one and work on it. On this main PR, though, let's stick to reviewing what's explicitly stated on the design doc.